app/models/ems_refresh/link_inventory.rb

```diff
@@ -1,138 +1,4 @@
 module EmsRefresh::LinkInventory
-  # Link EMS inventory through the relationships table
-  def link_ems_inventory(ems, target, prev_relats, new_relats)
```

Comment:
🎉 🎉 🎉 🎉

Reply:
Yeah, this and metadata_relats felt particularly good